client/src/constants/Route.ts

```diff
@@ -8,6 +10,8 @@ const routes_ = {
     game: (sessionId: string) => baseRoutes_.game.replace(":sessionId", sessionId)
 };
 
-
+export const redirectToLoginUrl = () => {
+    window.location.replace(`${config.baseUri}/${config.login.uri}`);
```

Comment:
Does this change the URL in the browser to the API `/auth/login` URL?

Reply:
Yea it does.

Reply:
https://github.com/rohanshiva/gitgame/blob/a2647bab80dc3692ca3e0915b65f55b4a86d7fbe/client/src/config/Config.ts#L2

https://github.com/rohanshiva/gitgame/blob/a2647bab80dc3692ca3e0915b65f55b4a86d7fbe/client/src/config/Config.ts#L13-L15

Locally this would go to `http://127.0.0.1:8001/auth/login`. 

And in prod I believe it goes to `https://gitgame.live/api/auth/login`